ddtrace/context.py

```diff
@@ -27,6 +33,7 @@
         _MetricDictType,  # _metrics
     ]
 
+INVALID_TRACESTATE_TAG_VALUE_CHARS = r",|;|:|[^\x20-\x7E]+"
```

Comment:
small nit: 
We should document this regex so it's easier to maintain. I renamed the constant but the original name is fine too.
```suggestion
# regex matches characters outside the ASCII range 0x20 to 0x7E
TRACESTATE_INVALID_CHARS_REGEX = r",|;|:|[^\x20-\x7E]+"
```